back/db/structure.sql

```diff
@@ -1773,7 +1773,7 @@ UNION ALL
  SELECT pr.id,
     pr.user_id AS dimension_user_id,
     COALESCE((pr.user_id)::text, (pr.id)::text) AS participant_id,
-    COALESCE(p.project_id, pr.phase_id) AS dimension_project_id,
+    p.project_id AS dimension_project_id,
```

Comment:
I noticed this before and it seems like a mistake. I confirmed with James and he told me I could change it like this